skimage/draw/draw.py

```diff
@@ -693,3 +693,75 @@ def bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape=None):
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=uint8)
     """
     return _bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape)
+
+
+def rectangle(start, end=None, extent=None, shape=None):
+    """Generate coordinates of pixels within an nd-rectangle. Either
+    end or extent are required.
+
+    Parameters
+    ----------
+    start : tuple
+        Origin point of the rectangle, ([plane], row, column[, ...])
+    end : tuple
+        End point of the rectangle ([num_planes], num_rows, num_cols[, ...]).
```

Comment:
This should probably be `([plane], row, column[, ...])`.

@jni, is this the standard way of specifying coords in N-dimensions?  I find it pretty confusing.